Task6.1/Task6.1/Program.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task6._1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите целое число = ");
+            int x;
+            x = int.Parse(Console.ReadLine());
+            Console.WriteLine(multipleornot(x));
+            Console.ReadKey();
+        }
+        static bool multipleornot(int number)
```

Comment:
Методы в C# принято называть с большой буквы и использовать camelCase
```suggestion
        static bool MultipleOrNot(int number)
```